python/__init__.py

```diff
@@ -347,7 +347,7 @@ def spi_connect(cls, serial, ignore_version=False):
     return None, handle, spi_serial, bootstub, None
 
   @classmethod
-  def usb_connect(cls, serial, claim=True):
+  def usb_connect(cls, serial, claim=True, wait=False):
```

Comment:
call it `silent`? wait doesn't mean anything in the context of this function

Reply:
I called it `no-error`